lib/jsonld-signatures.js

```diff
@@ -781,7 +781,7 @@ function _getDataToHash(input, options) {
     if(options.domain !== null && options.domain !== undefined) {
       toHash += '@' + options.domain;
     }
-  } else if(options.algorithm === 'LinkedDataSignature2015') {
+  } else if(options.algorithm === 'LinkedDataSignature2015' || options.algorithm === 'EcdsaKoblitzSignature2016') {
```

Comment:
I think we should change this to an `else` as it's the default for linked data signatures. Only `GraphSignature2012` has a special algorithm right now -- and using an `else` would future proof against this issue reoccurring with another algorithm.